cargo-dist/src/config/mod.rs

```diff
@@ -18,8 +18,8 @@ use crate::{
     METADATA_DIST,
 };
 
-mod version;
-pub use version::{get_version, ConfigVersion};
+pub mod version;
```

Comment:
we want to pub the mod and the stuff within it? that seems odd to me

Reply:
good catch -- made some minor changes so the mod can be private.